src/styles/styles.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import { pxToRem, mediaQuery } from '../styles';
+
+describe('Styles functions should return the correct values', () => {
+	it(' pxToRem function should return the correct number in rem', () => {
```

Comment:
`pxToRem` is a function and should be called `pxToRem()` => same for `mediaQuery`